phpcs.xml.dist

```diff
@@ -98,6 +98,12 @@
 	<rule ref="PHPCompatibility.Syntax.NewFunctionArrayDereferencing.Found">
 		<exclude-pattern>*/src/*.php$</exclude-pattern>
 	</rule>
+	<rule ref="PHPCompatibility.FunctionUse.NewFunctions.class_aliasFound">
```

Comment:
Please explain why those are excluded in `relevant technical choices` in PR message